### ISSUE TITLE
postinstall won't run in MINGW64

### ISSUE THREAD

**Issue by ianleckey** [syscolabs/kasaya#4]
```
> kasaya@0.1.1 postinstall C:\Users\SM\AppData\Roaming\npm\node_modules\kasaya
> ./scripts/postinstall.js

'.' is not recognized as an internal or external command,
operable program or batch file.
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! kasaya@0.1.1 postinstall: `./scripts/postinstall.js`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the kasaya@0.1.1 postinstall script.
```

Key bit being:
**'.' is not recognized as an internal or external command, operable program or batch file.**

Presumably just need to update package.json postinstall to `node scripts/postinstall.js`

**Comment by makdisse** [comment]
I had the same error. But when I ran the postinstall.js manually it worked fine. But kayasa won't run and I'm not sure where I can find logs to see why it is failing.

```
λ node postinstall.js
installing selenium and browser drivers. please wait...
installation is complete

λ kasaya
[...] Initializing Kasāya...

[x] Something went wrong :(, would you like to try again! :)
```

**Comment by AshanthaLahiru** [comment]
This cross-platform issue was fixed by PR #33 .
